### ISSUE TITLE
Exception: Failed to get GLTF data as array from accessor type SCALAR with semantic

### ISSUE THREAD

**Issue by znebby** [googlevr/poly-toolkit-unity#27]
We have a user that created some 3d models with Paint3D and uploaded them onto Google Poly. You can see them here: https://poly.google.com/search/Goodwins

They look okay in Poly on web, but when trying to import them, some of them give the error message in the title, while others are completely black (the user tried out exporting to different formats with Paint3D, glb and fbx).

**Comment by znebby** [comment]
I really hope we can get some updates on this soon.

Here are a couple more objects getting the Failed to get GLTF data error.

https://poly.google.com/view/3tPSEt8F9Ui (made in Blender)

https://poly.google.com/view/eRe7NA6fMD4 (made in 3DBuilder)

**Comment by znebby** [comment]
I have managed to fix this myself, however your project currently does not accept pull requests.

The issue is caused by the fact that you do not have support for Unsigned Int Component Types in GLTF Accessors. This was added in the GLTF2 spec.

If you change your mind, I will be happy to create a pull request with the fix.

**Comment by hanoixan** [comment]
I've seen this too. In lieu of a pull request, could you provide more information or a patch? Seems Poly isn't paying attention to this thread.

**Comment by znebby** [comment]
@hanoixan yeah I think I did the opposite, I converted the uint16 to uint32.

@dubois Now that there seems to be a Google team member active in the issues, it would be great to get a response on this.

**Comment by dubois** [comment]
The repo owner is aware of the issue but root cause is not yet known.

**Comment by znebby** [comment]
@dubois The root cause is quite simply that uint32 is not supported as a Component Type in GLTF Accessors.

**Comment by shohair** [comment]
@znebby Thankyou for the headsup, saved the day. 
"The issue is caused by the fact that you do not have support for Unsigned Int Component Types in GLTF Accessors. This was added in the GLTF2 spec."
and yes it really is quite simple.

**Comment by dagoch** [comment]
@znebby @hanoixan Could either of you share the source file and line number where you've made the fix, or the actual fixed code?  We're getting this error as well.  Thanks!
